apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -9,156 +14,251 @@ const svgProps: SvgIconProps = {
 };
 const iconProps = { svgSource: svgProps };
 
-const PrimaryTest: React.FunctionComponent = () => {
-  return (
-    <Notification
-      variant={'primary'}
-      action="Undo"
-      onPress={() => {
-        console.log('Notification tapped');
-      }}
-      onActionPress={() => {
-        console.log('Undo tapped');
-      }}
-    >
-      Mail Archived
-    </Notification>
-  );
-};
+const toastShowDuration = 600;
+const barShowDuration = 300;
+const hideDuration = 250;
+const toastBounciness = 1.5;
```

Comment:
nit: remake toast* to notification* ? `toastShowDuration -> notificationShowDuration`.
Also I think for constants, we usually use ALL_CAPS? So, `NOTIFICATION_SHOW_DURATION`?

Reason for name change is that "toast" is another name for notification, kind of an "alternate" name.
[Open UI](https://open-ui.org/components/toast.research) mentioned that can be named both. And it's actually the name of a native Android control as well (https://developer.android.com/guide/topics/ui/notifiers/toasts). To better distinguish ours / keep consistency, I think we should avoid the word toast :)